package/src/components/AttachmentPicker/AttachmentPicker.tsx

```diff
@@ -194,11 +194,11 @@ const renderImage = ({
     <AttachmentImage
       ImageOverlaySelectedComponent={ImageOverlaySelectedComponent}
       numberOfAttachmentPickerImageColumns={numberOfAttachmentPickerImageColumns}
-      onPress={isImage ? onPressImage : onPressVideo}
+      onPress={isImage === 'image' ? onPressImage : onPressVideo}
```

Comment:
is `isImage` a string? If so, the naming is pretty confusing. `isImage` makes it sound like it's a boolean. Could we rename it to be clearer as to what the variable holds?

Reply:
Solved in that PR. If that is merged it will be solved here to as this is created over that branch.